### DISCUSSION TITLE
TypeScript 5 decorators

### DISCUSSION THREAD

**Discussion by jdms754** [alex8088/electron-vite#418] [Q&A]
I looked through https://electron-vite.org/guide/typescript-decorator, which describes support for experimental TypeScript decorators. Are TypeScript 5 decorators, which are out of experimental phase, also supported?

**Comment by alex8088** [comment]
Yes, supported, actually supported by swc. You can also learn more by playing with the [example](https://github.com/alex8088/electron-vite-decorator-example).
